app/scripts/modules/google/loadBalancer/details/deleteModal/deleteModal.controller.ts

```diff
@@ -75,12 +83,16 @@ class DeleteLoadBalancerModalController implements ng.IComponentController {
       };
     } else {
       return () => {
-        return this.loadBalancerWriter.deleteLoadBalancer(this.loadBalancer, this.application, {
+        const command: IGoogleLoadBalancerDeleteOperation = {
+          cloudProvider: 'gce',
           loadBalancerName: this.loadBalancer.name,
+          accountName: this.loadBalancer.accountName,
```

Comment:
I should have `loadBalancer` be some base google load balancer interface here. In any case, the different load balancers all have `account`, but not `accountName`.